charts/incubator/tasmobackup/questions.yaml

```diff
@@ -25,22 +25,19 @@ questions:
           description: "Container Variable DBTYPE"
           schema:
             type: string
+            required: true
             default: "sqlite"
         - variable: DBNAME
           label: "DBNAME"
           description: "Container Variable DBNAME"
           schema:
             type: string
+            required: true
             default: "data/tasmobackup"
```

Comment:
If it supports a real database is should be setup with one.
If its only sqlite, there is no point of having those values shown in the UI.
@xstar97 Take a look please

Reply:
So MySQL or sqlite?

Reply:
If it works with mariadb, setup with one.
if only mysql, set to sqlite and remove both variables from UI.